server/mediaserver_test.go

```diff
@@ -1161,3 +1161,11 @@ func TestJsonProfileToVideoProfiles(t *testing.T) {
 	assert.Nil(p)
 	assert.Equal(common.ErrProfName, err)
 }
+
+func mustParseUrl(str string) *url.URL {
+	url, err := url.Parse(str)
+	if err != nil {
+		panic(err)
```

Comment:
Nit: I think it would be slightly better not to panic during tests, even if this particular case should never be reached, because a panic will cause all tests, not just the current one being run, to stop. If we use [T.Fatalf](https://golang.org/pkg/testing/#T.Fatalf) from the `testing` package here instead then we can immediately mark the test that this function is run within as a failure and exit the test but allow the next tests to run.

Ex.

```golang
func mustParseUrl(t *testing.T, str string) *url.URL {
    ...
    if err != nil {
        t.Fatalf(err)
    }
}
```